### PR TITLE
Front(feat): analytics settings [MARXAN-608]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

- Add script 
- Add events:
- [x] Sign up
- [x] New project
- [x] New scenario

- We still need to set two events when features were working:
- [ ] Download project
- [ ] Run scenario

### Testing instructions

[Plausible Analytics dashboard](https://plausible.io/marxan.vercel.app)

### Feature relevant tickets

[MARXAN-608](https://vizzuality.atlassian.net/secure/RapidBoard.jspa?rapidView=38&projectKey=MARXAN&modal=detail&selectedIssue=MARXAN-608&quickFilter=71)

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "next-compose-plugins": "^2.2.1",
     "next-connect": "^0.9.1",
     "next-optimized-images": "^2.6.2",
+    "next-plausible": "^2.0.0",
```
Comment (anamontiaga) [inline app/package.json:67]: integrated with NextJS:
https://github.com/4lejandrito/next-plausible

Comment (mbarrenechea) [reply app/package.json:67]: we need to trust 4lejandrito...

**Review by mbarrenechea — APPROVED** [review]

**Review by mbarrenechea — APPROVED** [review]
We will add the rest of events once everything is ready. @anamontiaga  Can you create a Jira task to prevent forgetting this?

**Comment by anamontiaga** [comment]
> We will add the rest of events once everything is ready. @anamontiaga Can you create a Jira task to prevent forgetting this?

yes, i had already done!

**Pull request merged**